CMakeLists.txt

```diff
@@ -284,6 +290,9 @@ if(FIPS)
     set(TEST_FIPS_PROPERTY "-DFIPS=true")
 else()
     set(TEST_FIPS_PROPERTY "-DFIPS=false")
+endif()
+
+if(EXPERIMENTAL_FIPS OR (NOT FIPS))
```

Comment:
should we drop a comment here that this guard can be relaxed once CKDF changes make it onto the FIPS branch version that we track?

Reply:
[There is a similar comment on the Java side.](https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/src/com/amazon/corretto/crypto/provider/AmazonCorrettoCryptoProvider.java#L106). I'll add one comment here as well.